platforms/aws/giantnetes/main.tf

```diff
@@ -161,6 +161,7 @@ module "bastion" {
 
   arn_region             = var.arn_region
   aws_account            = var.aws_account
+  aws_cni_subnets     = var.aws_cni_subnets_v2
```

Comment:
`terraform fmt`  would be nice :trollface:

Reply:
![download (11)](https://user-images.githubusercontent.com/1071648/106579874-e097e100-6549-11eb-8ee2-263e66d2604e.png)

did that before your comment :)

Reply:
yeah saw it after posting :p I bet it was my slow badwitdh